src/Network.cpp

```diff
@@ -643,7 +643,7 @@ void convolve(size_t outputs,
 
     // Weight shape (output, input, filter_size, filter_size)
     // 96 22 3 3
-    // outputs[96,19x19] = weights[96,22x3x3] x col[22x3x3,19x19]
+    // outputs[96,BOARD_SIZExBOARD_SIZE] = weights[96,22x3x3] x col[22x3x3,BOARD_SIZExBOARD_SIZE]
```

Comment:
This was a an example with specific values, so this doesn't make a lot of sense now. (Also Zero has 18, not 22 inputs...)

Reply:
Same question, should I just remove the comment then or simply put back 19 in it?

Reply:
I'd put the 19x19 back in.